### PR TITLE
Note that templates and the generated boilerplate code is 0BSD-licensed

### PR DESCRIPTION
**Pull request by FlorianWilhelm**
## Purpose

This addresses the points as discussed under conversation #459.

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -160,6 +160,12 @@ How can I use PyScaffold if my project is nested within a larger repository, e.g
     In future versions of PyScaffold this will be much simpler as ``pyproject.toml`` will completely replace ``setup.py``.
 
 
+What is the license of the generated project scaffold? Is there anything I need to consider?
+    The source code of PyScaffold itself is MIT-licensed with the exception of the subpackage ``pyscaffold.templates``,
```
Comment (abravalheri) [inline docs/faq.rst:164]: This seem to include the Python code under `pyscaffold/templates/__init__.py` itself. Is that the intention?

I think the best would be making only the `.template` files 0BSD. (Not that it differs to much from MIT, but it is nice to have the PyScaffold authorship mentioned in the case someone copies the `__init__.py` code.

File: src/pyscaffold/templates/__init__.py [hunk 2]
```diff
@@ -1,5 +1,7 @@
 """
 Templates for all files of a project's scaffold
+
+All template files within this subpackage are licensed under the BSD 0-Clause license.
```
Comment (abravalheri) [inline src/pyscaffold/templates/__init__.py:4]: Should we add a clarification here? E.g.

> All template files (`*.template`) within this subpackage are licensed under the BSD 0-Clause license.

**Review by abravalheri — APPROVED** [review]
Looks very good to me, thank you very much.

I added a few comments about the `templates/__init__.py` file itself.

File: src/pyscaffold/templates/__init__.py (hunk 2 above)
Comment (abravalheri) [reply src/pyscaffold/templates/__init__.py:4]:
```suggestion
All template files (`*.template`) within this subpackage are licensed under the BSD 0-Clause license.
```

File: docs/faq.rst (hunk 1 above)
Comment (abravalheri) [reply docs/faq.rst:164]:
```suggestion
    The source code of PyScaffold itself is MIT-licensed with the exception of the `*.template` files under the ``pyscaffold.templates`` subpackage,
```

File: docs/faq.rst [hunk 3]
```diff
@@ -160,6 +160,12 @@ How can I use PyScaffold if my project is nested within a larger repository, e.g
...
+    which is licensed under the BSD 0-Clause license (0BSD). Thus, also the generated boilerplate code for your project
```
Comment (abravalheri) [inline docs/faq.rst:165]:
```suggestion
    which are licensed under the BSD 0-Clause license (0BSD). Thus, also the generated boilerplate code for your project
```

**Comment by FlorianWilhelm** [comment]
Thanks @abravalheri for your remarks, I merged them all.

**Pull request merged**